source/_components/lock.zwave.markdown

```diff
@@ -22,3 +22,14 @@ Z-Wave locks will expose three services under the lock domain to manage usercode
 | clear_usercode | Clears a usercode at code_slot X. Valid code_slots are 1-254, but max is defined by the lock. |
 | get_usercode | Get a usercode from the lock at code_slot. Valid code_slots are 1-254, but max is defined by the lock. |
 | set_usercode | Sets usercode to X at code_slot Y. Valid usercodes are at least 4 digits, and max defined by the lock. |
+
+If your lock does not update the status properly for all events but the notification gets updated to show the state change you may need to set the
+`use_notification_state` flag for your lock. For example, with an Entity ID for the lock of `lock.my_lock you would set the following in your
+`configuration.yaml`:
+
+{% configuration %}
```

Comment:
This doesn't look like a [configuration variable section](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration).

I think you wanted to use
 ```yaml
zwave:
  device_config:
    lock.my_lock:
      use_notification_state: true
 ```